csrankings-h.csv

```diff
@@ -90,7 +90,7 @@ Haipeng Zhang,ShanghaiTech University,https://sist.shanghaitech.edu.cn/sist_en/2
 Haiping Lu,University of Sheffield,https://www.sheffield.ac.uk/dcs/people/academic/haiping-lu,CArrsDMAAAAJ
 Hairong Qi 0001,University of Tennessee,http://www.eecs.utk.edu/people/faculty/hqi,GqnNG-kAAAAJ
 Haitao Jiang,Shandong University,http://www.cs.sdu.edu.cn/info/1089/2334.htm,NOSCHOLARPAGE
-Haitao Wang 0001,Utah State University,https://cs.usu.edu/people/haitaowang,1PyJ5VwAAAAJ
+Haitao Wang 0001,University of Utah,https://cs.usu.edu/people/haitaowang,NOSCHOLARPAGE

```

Comment:
Home page still says Utah State; LinkedIn and home page say the same. Please clarify.